### PR TITLE
containers.podman.podman_container: Add env_files param

### PR DESCRIPTION
**Pull request by ant-bl**
Add a env_files parameter to use several file to set environment variables.

### CODE REVIEW

File: plugins/module_utils/podman/podman_container_lib.py [hunk 1]
```diff
@@ -53,6 +53,7 @@
     entrypoint=dict(type='str'),
     env=dict(type='dict'),
     env_file=dict(type='path'),
```
Comment (sshnaidm) [inline plugins/module_utils/podman/podman_container_lib.py:55]: For example here to change "path" to list of paths

File: plugins/module_utils/podman/podman_container_lib.py [hunk 2]
```diff
@@ -370,6 +371,11 @@ def addparam_env(self, c):
     def addparam_env_file(self, c):
         return c + ['--env-file', self.params['env_file']]
```
Comment (sshnaidm) [inline plugins/module_utils/podman/podman_container_lib.py:372]: this function to remove and always treat env_file as a list

File: plugins/module_utils/podman/podman_container_lib.py [hunk 3]
```diff
@@ -53,6 +53,7 @@
     entrypoint=dict(type='str'),
     env=dict(type='dict'),
     env_file=dict(type='path'),
+    env_files=dict(type='list', elements='path'),
```
Comment (sshnaidm) [inline plugins/module_utils/podman/podman_container_lib.py:56]: this to make an alias to `env_file`

File: plugins/module_utils/podman/podman_container_lib.py [hunk 4]
```diff
@@ -370,6 +371,11 @@ def addparam_env(self, c):
     def addparam_env_file(self, c):
         return c + ['--env-file', self.params['env_file']]
 
+    def addparam_env_files(self, c):
+        for env_file in self.params['env_files']:
+            c += ['--env-file', env_file]
+        return c
+
```
Comment (sshnaidm) [inline plugins/module_utils/podman/podman_container_lib.py:378]: and this to rename to `addparam_env_files`

**Review by sshnaidm — COMMENTED** [review]
would it work?

**Comment by sshnaidm** [comment]
@ant-bl please rebase on master, fixed CI in #695 
Just a thought - can we maybe just convert `env_file` to list (with alias to env_files) and use it as usual?

**Comment by ant-bl** [comment]
> would it work?

When I tested it works fine

> @ant-bl please rebase on master, fixed CI in #695 

Done

**Comment by sshnaidm** [comment]
@ant-bl  Sorry, probably I didn't explain well. My idea was not to add additional `env_files` argument, but just reuse existing `env_file` parameter making it as list, not as `path` type:
```python
env_file=dict(type='list', elements='path'),
```
```yaml
  env_file:
    description:
      -...
    type: list
    elements: path
    alias:
      - env_files
```
```python
    def addparam_env_file(self, c):
        for env_file in self.params['env_files']:
            c += ['--env-file', env_file]
        return c
```
It will be still backward compatible and will allow to specify multiple env files now.
WDYT?

**Comment by ant-bl** [comment]
> @ant-bl Sorry, probably I didn't explain well. My idea was not to add additional `env_files` argument, but just reuse existing `env_file` parameter making it as list, not as `path` type:
> 
> ```python
> env_file=dict(type='list', elements='path'),
> ```
> 
> ```yaml
>   env_file:
>     description:
>       -...
>     type: list
>     elements: path
>     alias:
>       - env_files
> ```
> 
> ```python
>     def addparam_env_file(self, c):
>         for env_file in self.params['env_files']:
>             c += ['--env-file', env_file]
>         return c
> ```
> 
> It will be still backward compatible and will allow to specify multiple env files now. WDYT?

@sshnaidm Hello. For me it is OK. IMHO the only think against your solution  is that it will break the compatibility of the code. If it is not an issue, it is OK.

**Pull request closed without merging**